src/it/scala/com/weightwatchers/reactive/kinesis/common/KinesisSuite.scala

```diff
@@ -233,7 +228,7 @@ trait KinesisSuite
   private def createLeaseTable(applicationName: String): Unit = {
     val manager = new LeaseManager[KinesisClientLease](s"$applicationName-$TestStreamName",
                                                        dynamoClient,
-                                                       new KinesisClientLeaseSerializer())
+                                                       new KinesisClientLeaseSerializer(),BillingMode.PROVISIONED)
```

Comment:
Should this be hardcoded here?

Reply:
The billing mode is used during creation of the dynamoDB leasing table. 
This is a setup trait for integration tests only that uses localstack (billing mode does not play any role). Since it is mandatory I just added the default.

Do you have a proposal how to deal with it?

Reply:
Ah didn't pay attention to the fact that it's a test, nevermind then ;)